test/testcases/test_data_onboarding.py

```diff
@@ -44,3 +44,38 @@ def test_proto_data_onboarding(self, setup, test_scenario, test_input, expected)
                                           password=setup["splunk_password"])
         logger.info("Splunk received %s events in the last hour", len(events))
         assert len(events) == expected
+
+    @pytest.mark.parametrize("test_scenario, test_input, expected", [
+        ("date_format", "latest=1365209605.000", "2010-06-13T23:11:52.454+00:00"),
+    ])
+    def test_extracted_timestamp_data_onboarding_date_format(self, setup, test_scenario, test_input, expected):
+        logger.info("testing {0} input={1} expected={2} event(s)".format(test_scenario, test_input, expected))
```

Comment:
If we are Python 3.7+, we can use f-strings, here and everywhere in the code when it is applicable.

Reply:
Done. Updated the code using f-strings.Thanks.